src/web/tools/GestureHandlerDelegate.ts

```diff
@@ -8,10 +7,10 @@ export interface MeasureResult {
   height: number;
 }
 
-export interface GestureHandlerDelegate<T> {
+export interface GestureHandlerDelegate<T, S> {
```

Comment:
I would suggest more descriptive type parameters. Expensify does a nice job with prefixing every generic type with the `T` letter. So in here you'd get `TComponent` and `THandler` or something like that.

Reply:
Done in [dbd44e2](https://github.com/software-mansion/react-native-gesture-handler/pull/2783/commits/dbd44e21e969dc6e93f40ea031282b19175bd844).